.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,8 +8,8 @@ assignees: ''
 ---
 
 **Information**
-<!-- Do not bother opening an issue for version 0.9.2 or older, it is extremely outdated. Please update to the latest one and see if your issue persists. -->
-- Solaar version (`git describe --tags` if cloned from Solaar repository):
+<!-- Do not bother opening an issue for version 1.0 or older.  Please update to the latest version and see if your issue persists. -->
```

Comment:
1.0 sounds for me a bit overlapping with 1.0.2. So I guess it is better to be more explicit. 
What about: `for version 1.0.1 or older.` ?

Reply:
I can see that. How about "older than version 1.0.1"?

Reply:
Fine for me